starfish/experiment/experiment.py

```diff
@@ -89,11 +107,69 @@ def name(self) -> str:
     def image_types(self) -> Set[str]:
         return set(self._images.keys())
 
-    def get_image(self, item: str, crop_parameters: Optional[CropParameters]=None) -> ImageStack:
-        return ImageStack.from_tileset(self._images[item], crop_parameters)
+    def show_aligned_image_groups(self):
+        """
+        Describe the aligned subgroups for each Tileset in this FOV
+
+        ex.
+            {'primary': 'Group 0:  <starfish.ImageStack (8, 2, 1, 205, 405))>',
```

Comment:
There is an extra closed brace on this and all such lines. 
```suggestion
            {'primary': 'Group 0:  <starfish.ImageStack (8, 2, 1, 205, 405)>',
```